### DISCUSSION TITLE
The use of ÆØÅ ?

### DISCUSSION THREAD

**Discussion by alftester** [Volue-Public/energy-mesh-python#294] [Q&A] [answered]
Hi, We're just staring to test in HECO and has succsesfully run GetVersion. 
Together with @gurosatherHeco, I have two questions that emerged while trying the functionality. 

1. Is it allowed to include ÆØÅ in the path?
![image](https://user-images.githubusercontent.com/108273645/184342510-b52b45fb-f327-44a6-a8af-bb8d0ccaf5d4.png)

2. Who can we contact in order to find the correct paths and UUIDs for objects in Mesh?

**Comment by tnoczyns-volue** [comment, marked as answer]
Hi,

Nice to meet you.
* Yes, it is allowed to include special characters in the path. However, in case of an error (like invalid path) the special characters may not be properly printed in the exception message.
* Contact: @sophievolue. I also suggest going through Python SDK documentation, especially the [Mesh model](https://vigilant-eureka-e3845ca2.pages.github.io/mesh_modelling.html) and [subsection on paths/ids](https://vigilant-eureka-e3845ca2.pages.github.io/mesh_modelling.html#objects-and-attributes-paths).

In case of any further questions just let me know.
